tox.ini

```diff
@@ -1,54 +1,53 @@
 [tox]
-envlist = py38-django{32,42},quality,translations-django{32,42}
+envlist = py{38, 312}-,quality,translations-django{42}
```

Comment:
Dropping Django 3.2 support is a breaking change. Maybe we could keep `py38-django{32,42}, py312-django42` instead?

Reply:
Also this is an intermediate PR and once we test with python 3.12, we'll eventually drop support for python 3.8 as well.

Reply:
@UsamaSadiq, dropping Python 3.8 and Django 3.2 in the `edx-platform` is a good thing, as it has the named releases. However, external libraries (like this XBlock) are compatible with past releases of `edx-platform`, so dropping Django 3.2 would be a breaking change for Palm, and dropping Python 3.8 would be a breaking change for Quince. So far, we have ensured that any new features are compatible with older releases, as the community widely uses them. This way, we don't need to fork repos and maintain feature branches for individual releases.

If both Python 3.8 and Django 3.2 work correctly in the CI, are there any benefits of removing them? We can drop them any time when we need to introduce some backward-incompatible changes, but at the moment, keeping them means that there are only a few additional checks in the CI. What do you think about it?

Reply:
Also, I'd be happy to work on adding the Python 3.12 support (while keeping the backward compatibility with Django 3.2 and Python 3.8) as the maintainer of this repo if you'd like.

Reply:
Sure, we can keep the backwards compatibility for the time being if this helps in maintaining the library in a better way. 
As I mentioned above, we are only aiming for removing python 3.8 and django 3.2 because their support has either already ended or going to end soon. If we want to keep these for some time, we can do that as well to streamline the process.